modules/reitit-openapi/src/reitit/openapi.cljc

```diff
@@ -12,10 +12,10 @@
 (s/def ::tags (s/coll-of (s/or :keyword keyword? :string string?)))
 (s/def ::summary string?)
 (s/def ::description string?)
-(s/def ::content-types (s/coll-of string?))
+(s/def :openapi/content-types (s/coll-of string?))
```

Comment:
Should be separate keys for request & response content-types, right?

Reply:
Right, duh